src/components/Input/index.js

```diff
@@ -36,4 +37,18 @@ const Input = glamorous.input(
   })
 );
 
+const Input = ({ value, onChange }) => (
+  <InputText type="text" value={value} onChange={onChange} />
+);
+
+Input.defaultProps = {
+  value: undefined,
+  onChange: null
```

Comment:
maybe make this undefined rather than null as null in snapshots looks ugly

Reply:
in fact we can probably just export the InputText glamorous component and set a default prop of type to text, deleting the input component which is purely used as a wrapper other than type=text